terraform/examples/io500/config/config.sh

```diff
@@ -30,13 +30,28 @@ ID=""
 PREEMPTIBLE_INSTANCES="true"
 SSH_USER="daos-user"
 DAOS_ALLOW_INSECURE="false"
+DAOS_POOLS='[
+  {
+    label = "io500_pool"
+    size = "1TB"
+    tier_ratio = 3
+    properties = "reclaim:disabled"
+    containers = [
+      {
+        label = "io500_cont",
+        type = "POSIX",
+        properties = "rf:0"
+      }
+    ]
+  }
+]'
 
 # Server(s)
 DAOS_SERVER_INSTANCE_COUNT="1"
-DAOS_SERVER_MACHINE_TYPE=n2-highmem-32 # n2-custom-20-131072 n2-custom-40-262144 n2-highmem-32 n2-standard-2
+DAOS_SERVER_MACHINE_TYPE=n2-standard-16 # n2-custom-20-131072 n2-custom-40-262144 n2-highmem-32 n2-standard-2
```

Comment:
Please put io500 config changes into a separate ticket.

Reply:
Ok, I will move this changes to other PR.